docs/docs/1-getting-started/1-contributors/1-setting-up-server.mdx

```diff
@@ -23,3 +23,6 @@ version and unzip it into the `Consul.Test` folder.
 This creates a 1-server cluster that operates in "dev" mode (does not
 write data to disk) and listens on `127.0.0.1:8500`.
 
+Note:
+The version of Consul you download should match the version you're contributing to.
```

Comment:
This is rather confusing. Our library supports a very wide range of Consul versions, so there's no requirement to download a particular one. I think we could suggest downloading the latest available version of the Consul agent because it allows you to test all features (unless features were made obsolete or removed in newer versions).